src/Dflydev/FigCookies/SetCookie.php

```diff
@@ -227,38 +229,37 @@ public function __toString() : string
         return implode('; ', $cookieStringParts);
     }
 
-    public static function create(string $name, ?string $value = null) : self
+    public static function create(string $name, ?string $value = null): self
     {
-        return new static($name, $value);
+        return new self($name, $value);
```

Comment:
class is not final, means this is a behavior change

Reply:
Reverted.